src/ngraph_builder.cc

```diff
@@ -2422,6 +2496,129 @@ static Status TranslateQuantizeAndDequantizeV2Op(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map, std::string op_name) {
+  int axis_index;         // index for concat_axis input
+  int value_start_index;  // start index for N tensor inputs
+  auto num_of_tensors_to_concat = (op->num_inputs() - 1) / 3;
+
+  if (op_name == "QuantizedConcat") {
+    axis_index = 0;
+    value_start_index = 1;
+  } else if (op_name == "QuantizedConcatV2") {
+    axis_index = num_of_tensors_to_concat;
+    value_start_index = 0;
+  } else {
+    return errors::InvalidArgument(
+        "This helper function is only used for QuantizedConcat and "
+        "QuantizedConcatV2 ops");
+  }
+
+  auto collect_nodes = [&op, &ng_op_map](int start, int end,
+                                         ng::NodeVector* p_ng_args) {
+    for (int i = start; i < end; i++) {
+      shared_ptr<ng::Node> ng_arg;
+      TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, i, &ng_arg));
+      (*p_ng_args).push_back(ng_arg);
+    }
+    return Status::OK();
+  };
+
+  // Collect the N input tensors to concat with
+  ng::NodeVector ng_args;
+  if (op_name == "QuantizedConcat") {
+    TF_RETURN_IF_ERROR(collect_nodes(value_start_index,
+                                     num_of_tensors_to_concat + 1, &ng_args));
+  } else if (op_name == "QuantizedConcatV2") {
+    TF_RETURN_IF_ERROR(
+        collect_nodes(value_start_index, num_of_tensors_to_concat, &ng_args));
+  }
+
+  // Get the input concat_axis
+  std::vector<int64> tf_concat_axis_vec;
+  TF_RETURN_IF_ERROR(GetStaticInputVector(op, axis_index, static_input_map,
+                                          &tf_concat_axis_vec));
+
+  // QuantizedConcat doesn't have negative concat_axis
+  int64 concat_axis = tf_concat_axis_vec[0];
+
+  // Get input_mins and input_maxs
+  std::vector<float> all_mins(num_of_tensors_to_concat),
+      all_maxs(num_of_tensors_to_concat);
+
+  // Construct input parameters to ScaledQuantizedConcat op
+  ng::NodeVector ng_all_mins, ng_all_maxs;
+  std::vector<float> min_tmp, max_tmp;
+
+  // Collect the N input mins and input maxes
+  for (int idx = 0; idx < num_of_tensors_to_concat; idx++) {
+    TF_RETURN_IF_ERROR(GetStaticInputVector(
+        op, num_of_tensors_to_concat + 1 + idx, static_input_map, &min_tmp));
+    TF_RETURN_IF_ERROR(
+        GetStaticInputVector(op, 2 * num_of_tensors_to_concat + 1 + idx,
+                             static_input_map, &max_tmp));
+
+    all_mins[idx] = min_tmp[0];
+    all_maxs[idx] = max_tmp[0];
+
+    auto min_node =
+        make_shared<ng::op::Constant>(ng::element::f32, ng::Shape{}, min_tmp);
+    auto max_node =
+        make_shared<ng::op::Constant>(ng::element::f32, ng::Shape{}, max_tmp);
+
+    ng_all_mins.push_back(std::make_shared<ngraph::op::Reshape>(
+        min_node, ngraph::AxisVector{}, ngraph::Shape{1}));
+    ng_all_maxs.push_back(std::make_shared<ngraph::op::Reshape>(
+        max_node, ngraph::AxisVector{}, ngraph::Shape{1}));
```

Comment:
Can we avoid this extra reshape? (Bu supplying appropriate args in the `Constant` constructor above

Reply:
Tried that, using Constant with `ng::Shape{1}`, but I got error doing that way.